content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -248,17 +248,18 @@ However, the particular path specified in the custom recycler Pod template in th
 `volumes` part is replaced with the particular path of the volume that is being recycled.
 
 ### PersistentVolume deletion protection finalizer
-{{< feature-state for_k8s_version="v1.23" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.31" state="beta" >}}
 
 Finalizers can be added on a PersistentVolume to ensure that PersistentVolumes
 having `Delete` reclaim policy are deleted only after the backing storage are deleted.
 
-The newly introduced finalizers `kubernetes.io/pv-controller` and
-`external-provisioner.volume.kubernetes.io/finalizer`
-are only added to dynamically provisioned volumes.
+The newly introduced finalizers `external-provisioner.volume.kubernetes.io/finalizer`
```

Comment:
Please avoid "newly added".
You may want to clarify the kubernetes release since which the finalizer is added.

Reply:
Please see https://github.com/kubernetes/website/pull/47282